OS/DiskOS/Help/API_2

```diff
@@ -1,7 +1,7 @@
 \CAPI V2 - Since \LIKO-12\C V1.0.0\7
 
-\91.\7 The extra libraries are no longer available as globals, but instead are now loadable by calling `local lib = Library("library_name")`.
+\91.\7 Since the extra libraries are no longer available as globals, they are now loadable by calling 'local lib = Library("library_name")'.
```

Comment:
Backticks are used to properly format code in Markdown (which LIKO-12 uses for docs)

\`local\` becomes `local`

Honestly I'm not sure if this should remain with backticks (which would translate directly to markdown and later to the JSON format that will be used for the [API docs](https://github.com/LIKO-12/API-Documentation)) or if we want to replace it to just ticks which display correctly in LIKO-12.

I'd prefer the later since the docs will probably need to be rewritten to fit the JSON format anyway.

@boredom101 what do you think?